### PR TITLE
Update documentations to new staructure

### PR DESCRIPTION
**Pull request by sachinira**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,46 +13,48 @@ access Microsoft Cloud service resources. This version of the connector only sup
 information of a specific account (currently logged-in user).
  
 For more information, go to the module(s).
-- [ballerinax/microsoft.teams](https://docs.central.ballerina.io/ballerinax/microsoft.teams/0.1.0)
+- [microsoft.teams](teams/Module.md)
```
Comment (shaniR) [inline README.md:16]: Remove unnecessary spaces in the above paragraph

File: README.md [hunk 2]
```diff
@@ -13,46 +13,48 @@ access Microsoft Cloud service resources. This version of the connector only sup
...
-  * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
- 
-  * [OpenJDK](https://adoptopenjdk.net/)
- 
-       > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed
-       JDK.
+## Building from the source
+### Setting up the prerequisites
+1.  Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle JDK](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).
+   > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed
```
Comment (shaniR) [inline README.md:21]: This should be a step. Please refer to other connectors. 
If we need to have this as a note, please use markdown syntax.

File: README.md [hunk 3]
```diff
@@ -13,46 +13,48 @@ access Microsoft Cloud service resources. This version of the connector only sup
...
+   JDK.
  
 2. Download and install [Ballerina](https://ballerina.io/)
+
+## Building from the source
```
Comment (shaniR) [inline README.md:26]: This section is duplicated.

File: teams/Module.md [hunk 4]
```diff
@@ -1,43 +1,18 @@
 ## Overview
-Ballerina connector for Microsoft Teams is connecting to MS Teams platform API in Microsoft Graph via Ballerina 
-language easily. It provides capability to perform basic functionalities provided in MS Teams such as Sending messages, 
+Ballerina connector for Microsoft Teams is connecting to MS Teams platform API in Microsoft Graph via Ballerina language.
```
Comment (shaniR) [inline teams/Module.md:2]: Let's rephrase this as "Microsoft Teams connector provides the capability to connect to the MS Teams platform API in Microsoft Graph via [Ballerina](https://ballerina.io/). It allows you to perform basic functionalities provided in MS Teams such as sending messages, viewing messages, creating teams, channels, and chats, deleting and updating resources, etc., programmatically. "

File: teams/Module.md [hunk 5]
```diff
@@ -54,57 +29,15 @@ First, in a new browser, enter the below URL by replacing the `<CLIENT_ID>` with
...
-    clientSecret = <MS_CLIENT_SECRET>
-    ```
- 
+    
 ## Quickstart
-## Create a Channel and send messages
-### Step 1: Import MS Teams Package
+To use the MS Teams connector in your Ballerina application, update the .bal file as follows:
+### Step 1 - Import connector
 First, import the ballerinax/microsoft.teams module into the Ballerina project.
```
Comment (shaniR) [inline teams/Module.md:36]: Let's remove the word "First,"

**Review by shaniR — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sachinira) [reply README.md]: Removed thanks

File: teams/Module.md (hunk 4 above)
Comment (sachinira) [reply teams/Module.md:2]: Fixed this

File: teams/Module.md (hunk 5 above)
Comment (sachinira) [reply teams/Module.md]: Removed

File: README.md (hunk 1 above)
Comment (sachinira) [reply README.md:15]: Fixed

File: README.md (hunk 2 above)
Comment (sachinira) [reply README.md]: This is added as per the [common format agreed by the team](https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit#heading=h.96g5nsmszl71). The changes which will alter that format will be added in next release cycles

**Review by abeykoon — APPROVED** [review]

File: teams/client.bal [hunk 6]
```diff
@@ -16,17 +16,20 @@
 
 import ballerina/http;
 
-# Teams Client Object for executing drive operations.
+# This is connecting to the Microsoft Graph RESTful web API that enables you to access Microsoft Cloud service resources.
```
Comment (shaniR) [inline teams/client.bal:19]: The Microsoft Teams connector lets you connect to the Microsoft Graph RESTful web API. This enables you to access Microsoft Cloud Services resources.

File: teams/client.bal [hunk 7]
```diff
@@ -16,17 +16,20 @@
...
     label: "Microsoft Teams Client",
     iconPath: "MSTeamsLogo.svg"
 }
-public client class Client {
-    http:Client httpClient;
-
-    # Initializes the Microsoft Teams connector client endpoint.
+public isolated client class Client {
+    final http:Client httpClient;
+    # Gets invoked to initialize the `connector`.
```
Comment (shaniR) [inline teams/client.bal:28]: Initializes the connector.

File: teams/types.bal [hunk 8]
```diff
@@ -232,7 +232,7 @@ public type Message record {|
     *MessageBaseData;
 |};
 
-# Represents properties of the body of an item, such as a message, event or group post
+# Properties of the body of an item, such as a message, event or group post.
```
Comment (shaniR) [inline teams/types.bal:235]: event,
